lab.R

```diff
@@ -188,24 +247,46 @@ mob_df = read_csv(file.path('data', 'mobility.csv'))
 
 #' 1. *This is just one way we could get at the relationship between stay-at-home in June and the peak of the outbreak in July.  What are some other approaches we might take?*
 #' 
-#' 
+#' i would think we should be more granular. like looking at weekly averages instead of monthly. we could also look rate of change rather than total and see 2 week lag correlates well.
 #' 
 
 #' 2. *Construct a dataframe `parks_june` that reports the mean level of "parks" mobility for each county in June 2020.  Call the variable `parks`.  (Just so the automatic checks know where to look.)  Note that the Google data has a lot of gaps for the `parks` type.  Use the `na.rm = TRUE` argument in `mean()` to handle missing values, and then filter out missing values.  The final dataframe should have three columns: county name, FIPS code, and `parks`.  And it should have one row for each county in the mobility data for which we have an estimate for "parks".*
 #' 
 
+parks_june = mob_df %>%
+  filter(type=='parks',
+  date>='2020-06-01',
+  date<='2020-06-30') %>%
+  group_by(county, fips) %>% 
+  summarize(parks=mean(pct_diff, rm.na =TRUE)) %>% 
```

Comment:
Typo.  So `mean()` is not ignoring NAs and you end up filtering out more counties than intended. 
```suggestion
  summarize(parks=mean(pct_diff, na.rm =TRUE)) %>% 
```